airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/commands.py

```diff
@@ -122,7 +151,12 @@ def validate_environment(is_local: bool, use_remote_secrets: bool):
         "upgrade_base_image": "pipelines.airbyte_ci.connectors.upgrade_base_image.commands.upgrade_base_image",
     },
 )
-@click.option("--use-remote-secrets", default=True)  # specific to connectors
+@click.option(
+    "--use-remote-secrets/--use-local-secrets",
+    help="Use Airbyte GSM connector secrets or local connector secrets.",
+    type=bool,
+    default=None,
+)
```

Comment:
Just curious, what does the help for this flag look like? does it still say `BOOLEAN`, even when providing both flag options?

Reply:
No type is displayed on flags. I think it makes sense as users do not have to pass an input.
```
Options:
  --use-remote-secrets / --use-local-secrets
                                  Use Airbyte GSM connector secrets or local
                                  connector secrets.
```

Reply:
thanks, this looks great!